src/feedback/snackbar/__test__/snackbar.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * To run this test:
+ * npx jest ./src/navigation/mobileStepper/__test__/mobileStepper.test.js
```

Comment:
Should be `npx jest ./src/feedback/snackbar/__test__/snackbar.test.js`

Reply:
Fixed.

Reply:
Hmm, not fixed...?  Need to push?

Reply:
Looking better as of d22ab45.
Resolving.